config/config_test.go

```diff
@@ -0,0 +1,55 @@
+package config_test
+
+import (
+	"os"
+
+	"github.com/tidepool-org/platform/config"
+	"github.com/tidepool-org/platform/store"
+
+	. "github.com/tidepool-org/platform/Godeps/_workspace/src/github.com/onsi/ginkgo"
+	. "github.com/tidepool-org/platform/Godeps/_workspace/src/github.com/onsi/gomega"
+)
+
+var _ = Describe("Config", func() {
+
+	Describe("FromJson", func() {
+
+		It("loads the given config file", func() {
+			var mgoConfig store.MongoConfig
+			config.FromJSON(&mgoConfig, "mongo.json")
+			Expect(mgoConfig).To(Not(BeNil()))
+			Expect(mgoConfig.URL).To(Not(BeEmpty()))
+		})
+
+		It("returns error if the config doen't exist", func() {
+			var random interface{}
+			err := config.FromJSON(&random, "random.json")
+			Expect(random).To(BeNil())
+			Expect(err).ToNot(BeNil())
+			Expect(err.Error()).To(ContainSubstring("random.json: no such file or directory"))
+		})
+	})
+
+	Describe("FromEnv", func() {
+
+		It("loads the given config value from env", func() {
+			const platformKey, platformValue = "CONFIG_TEST", "yay I exist!"
```

Comment:
I prefer separate lines for variable declarations/definitions.  It is much clearer, particularly if there are more than a couple variables.